app/src/androidTest/java/com/example/evently/ViewNotificationsTest.java

```diff
@@ -0,0 +1,251 @@
+        notificationDB.storeNotification(templateNotification(3, Notification.Channel.Winners));
+        notificationDB.storeNotification(templateNotification(4, Notification.Channel.Winners));
+        notificationDB.storeNotification(templateNotification(5, Notification.Channel.Winners));
+        notificationDB.storeNotification(templateNotification(6, Notification.Channel.Winners));
+        notificationDB.storeNotification(templateNotification(7, Notification.Channel.Winners));
+        notificationDB.storeNotification(templateNotification(8, Notification.Channel.Winners));
+
+        // Some notifications for the losers channel (for a few events).
+        notificationDB.storeNotification(templateNotification(1, Notification.Channel.Losers));
+        notificationDB.storeNotification(templateNotification(2, Notification.Channel.Losers));
+        notificationDB.storeNotification(templateNotification(3, Notification.Channel.Losers));
+        notificationDB.storeNotification(templateNotification(7, Notification.Channel.Losers));
+
+        // And some notifications for the cancelled channel (for a few events).
+        notificationDB.storeNotification(templateNotification(1, Notification.Channel.Cancelled));
+        notificationDB.storeNotification(templateNotification(2, Notification.Channel.Cancelled));
+        notificationDB.storeNotification(templateNotification(3, Notification.Channel.Cancelled));
+        notificationDB.storeNotification(templateNotification(5, Notification.Channel.Cancelled));
+    }
+
+    @AfterClass
+    public static void tearDownNotifications() throws ExecutionException, InterruptedException {
+        Tasks.await(Tasks.whenAllSuccess(notificationDB.nuke(), eventsDB.nuke()));
+    }
+
+    @Test
+    public void expectNotification_all() throws InterruptedException {
+        // Any notifications sent to the All channel for participated event IDs should show up.
+        // See the setUpNotifications to figure out which notifications we're expecting here.
+        final var expectedNotifications = new Notification[] {
+            templateNotification(1, Notification.Channel.All),
+            templateNotification(3, Notification.Channel.All),
+            templateNotification(7, Notification.Channel.All)
+        };
+
+        // For each of the expected notifications, scroll to it and make sure it shows properly.
+        //        for (final var expectedNotification : expectedNotifications) {
+        //            assertRecyclerViewItem(
+        //                    R.id.notif_list,
+        //                    p(R.id.notif_title, expectedNotification.title()),
+        //                    p(R.id.notif_description, expectedNotification.description()));
+        //        }
+        onView(withId(R.id.notif_list)).check(matches(isDisplayed()));
+    }
+
+    @Test
+    public void expectNotification_winner() throws InterruptedException {
```

Comment:
@Alexander-b1 I basically would like your help with filling up these tests. But I feel obliged to first figure out the recycler view stuff and write one full test with it (see commented out code just above) since that has proven very painful.